recipes/fiji-ilastik/IlastikSetter.java

```diff
@@ -0,0 +1,8 @@
+import java.util.prefs.Preferences;
+
+public class IlastikSetter {
+	public static void main(String... args) {
+		Preferences prefs = Preferences.userRoot().node("org/ilastik/ilastik4ij/ui/IlastikOptions");
```

Comment:
This will alter the user-wide setting if I am correct (@NicoKiaru, please correct me if I am wrong).

Reply:
I agree with @NicoKiaru that we should probably solve this problem of setting the prefs somehow in the `ilastik4ij` plugin. I don't think we should add anything to the package that would change the configuration globally for the user.

I'm sorry that there is so much back and forth on it, but given this solution, I think it's better to not automatically set the location of the executable for now.

Reply:
Hi,
Coming back to this issue of changing config for user instead of changing config for conda environment. I decided to propose a change in the fiji recipe: https://github.com/bioconda/bioconda-recipes/pull/36035
In this PR I define a new path for prefs user. Therefore, when we will set the path for the ilastik binary it will be restricted to the conda enviroment.
Do you think I should also set the system pref path to a different or this does not make sense?

Reply:
The PR has been merged. Youhou